src/ui/server/shared-validation/full-address/index.ts

```diff
@@ -12,7 +12,7 @@ import { RequestBody, ErrorMessageObject } from '../../../types';
  * @param {String} fieldId: Field ID
  * @param {String} errorMessage: Error message message
  * @param {Object} errors: Other validation errors for the same form
- * @returns {Object} providedAndMaxLength
+ * @returns {Function} providedAndMaxLength
```

Comment:
We call function `providedAndMaxLength` but we don't return function itself, function return would look like `return providedAndMaxLength;`.

Return type might be `ValidationErrors` from '../../../types';

Reply:
Might be a misread, we do return `providedAndMaxLength`, on line 17

```js
return providedAndMaxLength( ...
```

Reply:
We have () on line 17, so we are calling the function and returning it's result, not function itself.

```
// return function
return Math.abs;

// return result of function, number
return Math.abs(-1);
```

Reply:
Good spot, understood. We had a few instances of this, i've updated them all so it's now e.g

```js
* @returns {ValidationErrors} fullAddressValidation
```

Commit => https://github.com/UK-Export-Finance/exip/pull/2144/commits/46fe277d1e658f38d4a28a2dc0d3d7e8f7bc0148